### PR TITLE
Book store styling

### PR DESCRIPTION
**Pull request by medsonmoombe**
### In this pull request the following were implemented

- I used the Bookstore design in [Zeplin](https://zeplin.io/).
- Used the information detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Books from './Books';
+import './Book.css';
```
Comment (keroloussamy) [inline src/components/Book.js]: - [ ] Kindly there's no file called `Book.css` in your project so it gives an error when I try to open it.
image

Comment (keroloussamy) [inline src/components/Book.js]: - [ ] Kindly when I tried to remove it, the project runs but there are issues with the styling, Kindly fix them or add the `Book.css` file.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @medsonmoombe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mre55 — APPROVED** [review]
**Hi @medsonmoombe,** 

# Status: APPROVED 🏆 🎉 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**